.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CheckStyle-IDEA">
+    <option name="configuration">
+      <map>
+        <entry key="active-configuration" value="PROJECT_RELATIVE:$PROJECT_DIR$/.mvn/checkstyle.xml:.mvn/checkstyle.xml" />
+        <entry key="checkstyle-version" value="8.22" />
```

Comment:
how are we sure this version is in sync with maven-checkstyle-plugin?

Reply:
I was wondering about it as well but I didn't find an alternative - I'm ok with removing this file altogether and adding a README entry how to set up checkstyle.